borbskeeper/ParseManager/BorbParseManager.m

```diff
@@ -128,4 +164,15 @@ + (void)signOutUser{
     }];
 }
 
++ (PFFileObject *)getPFFileFromImage: (UIImage * _Nullable)image {
+    if (!image) {
+        return nil;
+    }
+    NSData *imageData = UIImagePNGRepresentation(image);
+    if (!imageData) {
+        return nil;
+    }
+    return [PFFileObject fileObjectWithName:@"image.png" data:imageData];
```

Comment:
why are we using "image.png", is it safe if we get multiple images?

Reply:
I don't think it currently matters if we use "image.png" because the only way we are accessing images is when they are associated with something (i.e. user-> profile image). I would also imagine in the future if we were to implement photo verification for tasks posted to the social the images would be associated to every task/post. At least in the sense it's 'safe' because each associated object has a property for a single UIImage, not multiple so its overwritten when adding a new one. There is also no situation where we need to extract all images or filtering queries by name. But I do see why that could be a problem in other applications.